contracts/periphery/DMMRouter02.sol

```diff
@@ -165,6 +179,9 @@ contract DMMRouter02 is IDMMRouter02 {
         if (pool == address(0)) {
             pool = IDMMFactory(factory).createPool(tokenA, tokenB, ampBps);
         }
+        // if we add liquidity to an existing pools this is unamplifed pool
```

Comment:
if we add liquidity to an existing pool this is unamplified pool

Reply:
updated